veascan-web/src/components/TxCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import styled from "styled-components";
+import Copy from "tsx:../../public/tx-info/Copy.svg";
+import ArbitrumLogo from "tsx:../assets/svgs/tx-info/ArbitrumLogo.svg";
+import EthereumLogo from "tsx:../assets/svgs/tx-info/EthereumLogo.svg";
+
+interface TxCardProps {
+  title: string;
+  chain: string;
+  txHash: string;
+  timestamp: string;
+  caller: string;
+  extraFields?: {
+    [key: string]: string;
+  };
+}
+
+const StyledDiv = styled.div`
+  margin: 0px 32px;
+  width: auto;
+  .icon {
+    width: 16px;
+    height: 16px;
+    fill: none;
+    cursor: pointer;
+  }
+
+  .tx-header {
+    font-size: 16px;
+    line-height: 20px;
+    font-weight: 600;
+    font-family: "Oxanium";
+    margin: 36px 0px 24px;
+    color: ${({ theme }) => theme.color.pink} !important;
+    width: fit-content;
+  }
+  .tx-info-text {
+    font-size: 16px;
+    line-height: 20px;
+    font-weight: 400;
+    font-family: "Oxanium";
+    width: fit-content;
+  }
+  .tx-info-copyable {
+    color: #6cc5ff !important;
```

Comment:
Try to not hardcode colors, if this color is not in the theme file, let me know and I will add it.